spec/controllers/timeline_entries_controller_spec.rb

```diff
@@ -173,6 +173,31 @@
 
       expect(coronavirus_page.reload.timeline_entries.count).to eq(0)
     end
+
+    it "doesn't delete the timeline_entry if draft_updater fails" do
+      create(:timeline_entry, coronavirus_page: coronavirus_page, heading: "Amidala")
+
+      coronavirus_page.reload
+      expect(coronavirus_page.timeline_entries.count).to eq(2)
+      expect(coronavirus_page.timeline_entries.first.position).to eq(2)
+      expect(coronavirus_page.timeline_entries.first.heading).to eq("Skywalker")
+      expect(coronavirus_page.timeline_entries.second.position).to eq(1)
+      expect(coronavirus_page.timeline_entries.second.heading).to eq("Amidala")
+
+      stub_publishing_api_isnt_available
+
+      delete :destroy,
+             params: {
+               id: timeline_entry.id,
+               coronavirus_page_slug: coronavirus_page.slug,
+             }
+
+      expect(coronavirus_page.timeline_entries.count).to eq(2)
+      expect(coronavirus_page.timeline_entries.first.position).to eq(2)
+      expect(coronavirus_page.timeline_entries.first.heading).to eq("Skywalker")
+      expect(coronavirus_page.timeline_entries.second.position).to eq(1)
+      expect(coronavirus_page.timeline_entries.second.heading).to eq("Amidala")
```

Comment:
This seems to be asserting on a bunch of things outside of this test and the logic in the code. 

You could probably simplify it down to:

```suggestion
      stub_publishing_api_isnt_available

      params = { id: timeline_entry.id, coronavirus_page_slug: coronavirus_page.slug }
      expect { delete :destroy, params: params }
         .not_to change { coronavirus_page.reload.timeline_entries.count }
         .from(2)
```

Reply:
Ah, I was trying to assert that the position doesn't change, as if the timeline entry was being recreated rather than rolled back, the positions would reverse order.